### DISCUSSION TITLE
ADMIN_TOKEN docs

### DISCUSSION THREAD

**Discussion by Daniel15** [dani-garcia/vaultwarden#1824] [Q&A]
The docs at https://github.com/dani-garcia/vaultwarden/wiki/Using-the-MariaDB-%28MySQL%29-Backend show this environment variable:
```
ADMIN_TOKEN=<some_random_token_as_per_above_explanation>
```

But there is no "above explanation" on the page 🤔 I didn't see any explanation on https://github.com/dani-garcia/vaultwarden/wiki/Starting-a-Container or https://github.com/dani-garcia/vaultwarden/wiki/Using-Docker-Compose either.

Did this 'explanation' go missing at some point?

**Comment by BlackDex** [comment]
See: https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page
